packages/pharos/src/components/button/PharosButton.react.stories.jsx

```diff
@@ -54,7 +54,7 @@ export const Base = {
 };
 
 export const Variants = {
-  render: (_) => (
+  render: () => (
```

Comment:
I had previously kept these because without them Storybook wouldn't render a viewable code snippet in the **Docs** tab; it appears to be now, though.